frontend/src/views/Combat.vue

```diff
@@ -278,16 +278,18 @@ export default {
         return;
       }
 
+      this.waitingResults = true;
+
       // Force a quick refresh of targets
       await this.fetchTargets({ characterId: this.currentCharacterId, weaponId: this.selectedWeaponId });
       // If the targets list no longer contains the chosen target, return so a new target can be chosen
       if (!this.targets.find((target) => target.original === targetToFight.original)) {
+        this.waitingResults = false;
```

Comment:
this still needs to be set outside the return too

Reply:
I think that would enable the buttons again before battle finishes. I never checked what sets it back to false if you battle but it probably has to do with getting fight results back. Not home right now, will check and update when I can.

Reply:
@bilbolPrime let me know the fruits of your research when you can!

Reply:
Hello!
Sorry for delay, hectic time at work
So the reason I never gave this much attention when I did that change was that the method never set it back to false (after await) so I just figured that some callback set it to false. And it's being done in this method:
`async updateResults([fightResults, error]) {
      this.resultsAvailable = fightResults !== null;
      this.waitingResults = fightResults === null && error === null;
      this.setIsInCombat(this.waitingResults);
      if (this.resultsAvailable) this.$bvModal.show('fightResultsModal');
    }`

When the battle is over or an error is thrown that method sets waitingResults to false again. If there is an issue in relying on that method, then it's on prod right now even without this change. updateResults is being watched so I am guessing it's being called whenever fightResults or error changes. The change should not introduce any issue that doesn't already exist and I don't think there is.

Reply:
Sounds good to me, thanks for the analysis!